src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.prevalues.controller.js

```diff
@@ -29,22 +29,21 @@ angular.module("umbraco").controller("Umbraco.PrevalueEditors.RteController",
                 // extend commands with properties for font-icon and if it is a custom command
                 $scope.tinyMceConfig.commands = _.map($scope.tinyMceConfig.commands,
                     function(obj) {
-                        var icon = getFontIcon(obj.alias);
-                        var iconName = "";
-                        var svgName = "";
-                        var testName = icon.name.startsWith("icon-") ? icon.name : "icon-" + icon.name;
-                        if (data.some(e => e.name === testName)) {
-                            svgName = testName;
-                        } else {
-                            iconName = "mce-ico " + (icon.isCustom ? " mce-i-custom " : " mce-i-") + icon.name;
-                        }
+                        var fontIcon = getFontIcon(obj.alias);
+                        var isCustom = fontIcon.isCustom;
+
+                        var iconName = fontIcon.name.startsWith("icon-") ? fontIcon.name : "icon-" + fontIcon.name;
+
+                        var icon = data.some(e => e.name === iconName)
+                            ? iconName
+                            : "mce-ico " + (isCustom ? "mce-i-custom " : "mce-i-") + fontIcon.name;
+
                         var objCmd = Utilities.extend(obj,
                             {
                                 fontIcon: icon.name,
                                 isCustom: icon.isCustom,
                                 selected: $scope.model.value.toolbar.indexOf(obj.alias) >= 0,
-                                icon: iconName,
-                                svg: svgName
+                                icon: icon
```

Comment:
@RachBreeze you can probably also remove the `fontIcon` property here since we don't use this and `icon` variable is now a string, so `isCustom` property should use `isCustom` variable defined above.